app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -10,6 +10,8 @@ def push(event:, options: {})
       # This is done synchronously before the rest of the publishing.
       # Currently (02/11/2016) publishing-api links
       # are not recalculated on parent documents when their translations are unpublished.
```

Comment:
I think this comment specifically relates to the `handle_translations` lines.

Reply:
Ah, I will move the raise above it 👍